libparsec/crates/platform_device_loader/tests/recovery.rs

```diff
@@ -1,3 +1,43 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-// TODO !
+#![allow(clippy::unwrap_used)]
+
+use libparsec_platform_device_loader::{export_recovery_device, import_recovery_device};
+use libparsec_tests_lite::parsec_test;
+use libparsec_types::HumanHandle;
+use libparsec_types::ParsecOrganizationAddr;
+use libparsec_types::UserProfile;
+use libparsec_types::{DeviceLabel, LocalDevice};
+
+#[parsec_test]
+async fn test_ok_recovery() {
+    let url = ParsecOrganizationAddr::from_any(
+        // cspell:disable-next-line
+        "parsec3://127.0.0.1:6770/Org?no_ssl=true&p=xCD7SjlysFv3d4mTkRu-ZddRjIZPGraSjUnoOHT9s8rmLA",
```

Comment:
Don't use a real address if we don't plan on actually reaching it (otherwise this can lead to unexpected things if our code is buggy and the url is actually accessed)

Instead you should use the `.invalid` top level domain [that is a reserved TLD for this kind of needs](https://en.wikipedia.org/wiki/.invalid), so typically `parsec3://parsec.invalid/Org?...` (also note we don't need `no_ssl=true`)